### DISCUSSION TITLE
Invalid credentials

### DISCUSSION THREAD

**Discussion by tomasenskede** [photoprism/photoprism#4476] [Q&A] [answered]
I change password on the admin user... then I cant login again... I get error "Invalid credentials"... 

I have reseted the password using  "photoprism passwd admin", same error

I have created an new user using "photoprism users add", when trying to logon with the new users I get th  same error

Whats broken?

**Comment by lastzero** [comment]
https://docs.photoprism.app/getting-started/troubleshooting/#cannot-log-in

**Reply by tomasenskede** [reply]
"Remember that the initial admin username and password cannot be changed after PhotoPrism has been started for the first time"

??

**Reply by lastzero** [reply]
You can change it, just not the initial values in the config after they have already been applied on first startup. That would require time travel.

**Reply by tomasenskede** [reply]
sorry, I dont get it... is clean install the only option to get in controll again?

**Reply by lastzero** [reply]
No, please read the full documentation.

**Reply by tomasenskede** [reply]
I have, I haven't checked the communication with db... as the only action I did was to update the admin-password in the gui... since then I am not able to login.

**Reply by lastzero** [reply]
https://docs.photoprism.app/user-guide/users/cli/

**Reply by tomasenskede** [reply]
root@photoprism:/opt/photoprism# photoprism passwd admin
INFO[2024-08-24T08:24:36+02:00] Become a member today, support our mission and enjoy our member benefits! 💎 
INFO[2024-08-24T08:24:36+02:00] Visit https://www.photoprism.app/membership to learn more. 
WARN[2024-08-24T08:24:36+02:00] **config: password required to initialize 'admin' account** 
INFO[2024-08-24T08:24:36+02:00] please enter a new password for admin (8-72 characters) 
New Password: 
Retype Password: 
INFO[2024-08-24T08:24:48+02:00] **password for admin has been changed**          
root@photoprism:/opt/photoprism#

**Reply by tomasenskede** [reply, marked as answer]
After clean install

![image](https://github.com/user-attachments/assets/62c2a4a2-811b-4561-85c7-3c7392ca8f83)

And now it works! I finally have a fully functional PhotoPrism instance with a secure admin password.
